src/components/home/Modal.tsx

```
@@ -25,15 +30,31 @@ const HomeModal = ({ setIsOpen, setIsChecked, isChecked }: Props) => {
     };
   }, [setIsOpen]);
 
+  useEffect(() => {
+    const fetchSurvey = async () => {
+      const survey = await getSurvey();
+      setSurvey(survey);
+    };
+    fetchSurvey();
+  }, []);
+
+  const handleCheck = async () => {
```

ESLint

[error] 'handleCheck' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)